dynamic_programming/tribonacci.py

```diff
@@ -0,0 +1,25 @@
+# Tribonacci sequence using Dynamic Programming
+
+
+def tribonacci(num: int) -> list:
```

Comment:
```suggestion
def tribonacci(num: int) -> list[int]:
```
Type hint the list elements